src/common/enums/skill/skill-type.ts

```diff
@@ -0,0 +1,7 @@
+enum SkillType { // TODO: real values from server
+  SOFT_SKILLS,
+  HARD_SKILLS,
+  OTHER,
+}
```

Comment:
And also remove helper for numeric enums
```suggestion
enum SkillType { // TODO: real values from server
  SOFT_SKILLS = 'softSkills',
  HARD_SKILLS = 'hardSkills',
  OTHER = 'other',
}
```